### DISCUSSION TITLE
usePreinstalledWDA on ios simulator

### DISCUSSION THREAD

**Discussion by Auto81** [appium/appium#18792] [Q&A]
is there anyway to stop the WDA being reinstalled everytime on an ios simulator between tests?

I have tried usePreinstalledWDA, it's only supported for real devices

**Comment by KazuCocoa** [comment]
Afaik, no for simulators right now. `xcrun  simctl launch` cannot use for the WDA usage, so if someone figures out the method, we can support simulators as well
